### ISSUE TITLE
More flexible type encoding in `multipart_eml_parsing`

### ISSUE THREAD

**Issue by JM-Lemmi** [cirosec-studis/python-zimbra-web#55]
The `Content-Transfer-Encoding` is also "quoted-printable" so you might need to decode into utf8 binary before setting it as body if that is the case:
```python
>>> import quopri
>>> mystring = "Testmail mit 2 Anh=C3=A4ngen und Text"
>>> decoded = quopri.decodestring(mystring).decode("utf-8")
>>> decoded
'Testmail mit 2 Anhängen und Text'
```

_Originally posted by @frereit in https://github.com/cirosec-studis/python-zimbra-web/pull/54#discussion_r745875936_

This will always base64 decode, even if the `Content-Transfer-Encoding` is something else (`binary`, `quoted-printable`, `8bit`, `7bit`). Maybe at least check if the `Content-Transfer-Encoding` is base64, otherwise just assume it's raw. `quoted-printable` is not really used for attachments.

_Originally posted by @frereit in https://github.com/cirosec-studis/python-zimbra-web/pull/54#discussion_r745871981_

**Comment by frereit** [comment]
Fixed by #61 as we now use `.get_payload(decode=True)` so that we just get the decoded data and don't have to worry about any transfer encodings :)

**Comment by JM-Lemmi** [comment]
Man, the email.parser package is really elaborate and useful actually
